content/ecosystem/clients/tammy.md

```diff
@@ -0,0 +1,29 @@
++++
+title = "Tammy"
+template = "ecosystem/client.html"
+[extra]
+thumbnail = "tammy.png"
+maintainer = "connect2x GmbH"
+maturity = "Stable"
+repo = "https://gitlab.com/connect2x/tammy"
+matrix_room = "#tammy:imbitbu.de"
+licence = "Apache-2.0"
+latest_release = "2024-11-01"
+featured = true
```

Comment:
```suggestion
featured = false
```

The featured clients are usually the ones that support more Matrix features, such as voip and threads. We will be happy to revisit it once Tammy supports it!